src/main/java/org/jabref/gui/filelist/FileListDialogController.java

```diff
@@ -0,0 +1,157 @@
+package org.jabref.gui.filelist;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import javax.inject.Inject;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.TextField;
+
+import org.jabref.Globals;
+import org.jabref.gui.AbstractController;
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.desktop.JabRefDesktop;
+import org.jabref.gui.externalfiletype.ExternalFileType;
+import org.jabref.gui.externalfiletype.ExternalFileTypes;
+import org.jabref.gui.externalfiletype.UnknownExternalFileType;
+import org.jabref.gui.util.FileDialogConfiguration;
+import org.jabref.logic.util.io.FileUtil;
+import org.jabref.model.entry.LinkedFile;
+import org.jabref.model.util.FileHelper;
+import org.jabref.preferences.JabRefPreferences;
+import org.jabref.preferences.PreferencesService;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class FileListDialogController extends AbstractController<FileListDialogViewModel> {
+
+    private static final Log LOGGER = LogFactory.getLog(FileListDialogController.class);
+    private static final Pattern REMOTE_LINK_PATTERN = Pattern.compile("[a-z]+://.*");
+
+    @FXML private TextField tfLink;
+    @FXML private Button btnBrowse;
+    @FXML private Button btnOpen;
+    @FXML private TextField tfDescription;
+    @FXML private ComboBox<ExternalFileType> cmbFileType;
+    @FXML private Button btnOk;
+    @FXML private Button btnCancel;
+
+    @Inject private PreferencesService preferences;
+    @Inject private DialogService dialogService;
+    @Inject private StateManager stateManager;
+
+    private boolean showSaveDialog;
+
+    @FXML
+    private void initialize() {
+        viewModel = new FileListDialogViewModel(stateManager.getActiveDatabase().get());
+        setBindings();
+
+    }
+
+    private void setBindings() {
+
+        cmbFileType.itemsProperty().bindBidirectional(viewModel.externalFileTypeProperty());
```

Comment:
@tobiasdiez  Here the binding is created, a simple list property for the combobox

Reply:
this looks good! (a one-directional binding should suffice I think since it looks like the `itemsProperty` is never changed directly.)